lib/Ast_to_Mir.ml

```diff
@@ -61,7 +73,7 @@ and trans_exprs = List.map ~f:trans_expr
 let trans_sizedtype = map_sizedtype trans_expr
 
 let neg_inf =
-  { expr= FunApp (string_of_internal_fn FnNegInf, [])
+  { expr= FunApp (CompilerInternal, string_of_internal_fn FnNegInf, [])
```

Comment:
Hmm, this negative_infinity() is actually math library function... but on the other hand FnNegInf is defined in the MIR as one of these compiler internals. A small part of me wants to just mark this as StanLib, but not sure if that'd have weird repercussions.